src/stravalib/client.py

```diff
@@ -588,6 +588,38 @@ def get_club_activities(
             limit=limit,
         )
 
+    def get_club_admins(
```

Comment:
can you please test this one out? i'm not a club admin myself so i'm not sure if i have access to this via the clubs i'm in. 

i tried to call it and i don't get anything back which is odd. 
```
all_admins = client.get_club_admins(club_id)
for admin in all_admins:
    print(admin)
``` 

i don't have time to troubleshoot right now but it could just be a permissions thing. if so we might want to adjust the docstring.

Reply:
I'm getting a normal response with an iterator of `SummaryAthlete` objects, regardless of whether I'm an admin or not in that club.